fireapp/lib/main.dart

```diff
@@ -59,6 +60,7 @@ class _MyAppState extends State<MyApp> {
         '/volunteer_list': (context) => const VolunteerList(),
         '/dietary_requirements/update': (context) => const DietaryRequirementsPage(),
         '/server_url_page': (context) => const ServerUrlPage(),
+        '/shift_request': (context) => const ShiftRequestView(requestId: "1")
```

Comment:
Good for now, but once @naychithanshwe has finished her work, we'll need to not hardcode the parameter